ipa-core/tests/common/mod.rs

```diff
@@ -24,6 +24,7 @@ pub mod tempdir;
 pub const HELPER_BIN: &str = env!("CARGO_BIN_EXE_helper");
 pub const TEST_MPC_BIN: &str = env!("CARGO_BIN_EXE_test_mpc");
 pub const TEST_RC_BIN: &str = env!("CARGO_BIN_EXE_report_collector");
+pub const CRYPTO_UTIL_BIN: &str = env!("CARGO_BIN_EXE_crypto_util");
```

Comment:
it seems this works locally, but is not finding it in the github tests. is there something else I need to do to make sure this compiles for the tests?

Reply:
@andyleiserson do you know if I need to update something to get this binary to also compile?